src/platform/qt/ConfigController.cpp

```diff
@@ -171,6 +179,15 @@ ConfigController::ConfigController(QObject* parent)
 			self->m_argvOptions[optionName] = QString::fromUtf8(arg);
 			return true;
 		}
+#ifdef ENABLE_SCRIPTING
+		if (optionName == QLatin1String("script")) {
+			QString scriptName = QString::fromUtf8(arg);
+			QList<QString> currentList = self->m_argvOptions[optionName].toStringList();
+			currentList.push_back(scriptName);
+			self->m_argvOptions[optionName] = QVariant(currentList);
```

Comment:
Please follow the same convention as the ecard block so the option can be specified multiple times.